.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
With TSDX, we shouldn't necessarily need a custom `eslintrc`, since that's already provided for us by the library.